templates/display_product.tt.html

```diff
@@ -52,7 +52,8 @@ <h1 property="food:name" itemprop="name">[% title %]</h1>
 <!-- Display UPC code if the EAN starts with 0 -->
 [% IF upc_code == 'defined' %]
     <p id="barcode_paragraph">
-        [% lang("barcode") %]:  <span id="barcode" property="food:code" itemprop="gtin13" style="speak-as:digits;">[% code %]</span>[% upc %]
+        [% lang("barcode") %]: <p id="_barcode">[% code %]</p>
+      <svg id="barcode" property="food:code" itemprop="gtin13" style="speak-as:digits;"></svg>
```

Comment:
I'm not sure if the structured data, and `speak-as:digits` work on `svg` elements. Wouldn't the attributes be more useful on the `id=_barcode` element? Then, the SVG could be added dynamically in JavaScript, if it's required, so that there wouldn't be an empty dangling SVG.

Reply:
The JsBarcode library requires the element to exist inside dom and we just provide the id of that element to JsBarcode so, i don't think that creating svg dynamically would be a great idea, since the script tag is placed at the end of the body tag, so i dont think there is a chance of dangling svg.

Reply:
I thought there could be a dangling/empty SVG, if the user has JavaScript disabled.

Reply:
Ah, i didn't thought about that u are right 😅, i'll fix it, any other problem ?

Reply:
I have resolved the issues, can u review the updated commit ??

Reply:
Please can u approve the latest commit for merge, if there is no issue ?